### DISCUSSION TITLE
dataset creation

### DISCUSSION THREAD

**Discussion by mahewashabdi** [google/lightweight_mmm#22] [General]
Hi 

I have a question regarding the dataset. My dependent variable is Revenue and my indpendent var are bing, fb,google, and linkedin.. I am considering the cost spend and impression/clicks for these channels. To model the revenue corresponding to each day , do i need to take into account the  revenue generated by the organic channels because in that impressions and cost are 0 so i can add that as a separate feature but then do i need to consider that revenue in my dependent variable

**Comment by pabloduque0** [comment]
Hello @mahewashabdi ! Great question!

Short answer is yes, you can include your full revenue in your dependent variable. The model will estimate a "non media" baseline that accounts for the part of the revenue that is not coming from your media investment.

As for wether and how to include organic channels there are a few options, [refer to my other answer](https://github.com/google/lightweight_mmm/issues/4#issuecomment-1105433685) for some of them but let me know if there are any outstanding questions.

**Reply by mahewashabdi** [reply]
Thank you @pabloduque0
